templates/tfengine/recipes/folder.hcl

```diff
@@ -41,7 +41,7 @@ schema = {
 
 template "deployment" {
   recipe_path = "./deployment.hcl"
-  output_path = "{{.display_name}}/folder"
+  output_path = "./folder"
```

Comment:
does these change the generated files?

Reply:
No it should generate the same files because i updated the output_path to include this prefix manually

Reply:
so in this case the {{display_name}} was .?

Reply:
It was "prod"

If you look at the example, template "folder_prod" now has the output path `./live/prod`. Before it was `./live`.